src/Server/LspRequestRouter.cs

```diff
@@ -21,11 +21,11 @@ namespace OmniSharp.Extensions.LanguageServer.Server
 {
     internal class LspRequestRouter : RequestRouterBase<ILspHandlerDescriptor>, IRequestRouter<IHandlerDescriptor>
     {
-        private readonly IEnumerable<ILspHandlerDescriptor> _collection;
+        private readonly IHandlerCollection _collection;
         private readonly IEnumerable<IHandlerMatcher> _handlerMatchers;
 
         public LspRequestRouter(
-            IEnumerable<ILspHandlerDescriptor> collection,
```

Comment:
This was why descriptors could not be found with the latest beta... I need to add an integration test with the `LanguageServer` for this at somepoint.